stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/textinput/StreamMessageInputView.kt

```diff
@@ -0,0 +1,165 @@
+                R.styleable.StreamMessageInputView_attachmentButtonHeight,
+                context.resources.getDimensionPixelSize(R.dimen.stream_attachment_button_height)
+            )
+        }
+    }
+
+    private fun configLightningButton(typedArray: TypedArray) {
+        binding.ivOpenEmojis.run {
+            isVisible =
+                typedArray.getBoolean(R.styleable.StreamMessageInputView_showLightningButton, true)
+
+            typedArray.getDrawable(R.styleable.StreamMessageInputView_lightningButtonIcon)
+                ?.let(this::setImageDrawable)
+
+            DrawableCompat.setTintList(
+                drawable,
+                getColorList(
+                    typedArray.getColor(
+                        R.styleable.StreamMessageInputView_lightningButtonIconColor,
+                        ContextCompat.getColor(context, R.color.stream_gray_dark)
+                    ),
+                    typedArray.getColor(
+                        R.styleable.StreamMessageInputView_lightningButtonIconPressedColor,
+                        ContextCompat.getColor(context, R.color.stream_white)
+                    ),
+                    typedArray.getColor(
+                        R.styleable.StreamMessageInputView_lightningButtonIconPressedColor,
+                        ContextCompat.getColor(context, R.color.stream_gray_light)
+                    )
+                )
+            )
+
+            layoutParams.width = typedArray.getDimensionPixelSize(
+                R.styleable.StreamMessageInputView_lightningButtonWidth,
+                context.resources.getDimensionPixelSize(R.dimen.stream_attachment_button_width)
+            )
+
+            layoutParams.height = typedArray.getDimensionPixelSize(
+                R.styleable.StreamMessageInputView_lightningButtonHeight,
+                context.resources.getDimensionPixelSize(R.dimen.stream_attachment_button_height)
+            )
+        }
+    }
+
+    private fun showSendMessageEnabled() {
+        binding.ivSendMessage.setImageResource(R.drawable.stream_ic_filled_up_arrow)
+    }
+
+    private fun hideSendMessageEnabled() {
+        binding.ivSendMessage.setImageResource(R.drawable.stream_ic_filled_right_arrow)
+    }
+
+    private fun configTextInput(typedArray: TypedArray) {
+        TextInputHandler(binding.etMessageTextInput, ::showSendMessageEnabled, ::hideSendMessageEnabled)
+
+        binding.etMessageTextInput.run {
+            setTextColor(
+                typedArray.getColor(
+                    R.styleable.StreamMessageInputView_inputTextColor,
+                    ContextCompat.getColor(context, android.R.color.black)
+                )
+            )
+
+            setHintTextColor(
+                typedArray.getColor(
+                    R.styleable.StreamMessageInputView_inputHintTextColor,
+                    ContextCompat.getColor(context, android.R.color.darker_gray)
+                )
```

Comment:
The input hint is a light blue colour in the gif in the PR description. Is that just out of date? Is it a bug due to using a system colour here somehow, instead of one of our own? (Same goes for all `android.R.color` resources in this file, are we sure we want to rely on those?)

Reply:
You're right, I'll create the correct colour.

Reply:
By the way, the blue hint in blue is just because I was testing the customisation of the hint when a created that gift. Good catch haha.